app/views/start/index.html.erb

```diff
@@ -8,3 +8,26 @@
   <li>skills and qualifications required</li>
   <li>tasks and hours</li>
 </ul>
+ 
+<%= form_tag("/select_soc_codes", method: "get") do %>
+  <div class="form-group ">
+    <label for="id_sector_form-sector_0">
+      <span class="form-label">Work, skills and interests</span>
+    </label>
+    <ul class="multi-entry-text-input">
+    <li>
+      <%= text_field_tag(:keyword_1, nil, class: 'form-control') %>
```

Comment:
Plain HTML is easier to understand, I don't thin we should use rails helpers unless they're adding value (like handling validation, etc.)